packages/fxa-auth-server/lib/senders/templates/partials/postVerifyPrimary.html

```diff
@@ -0,0 +1,24 @@
+<tr style="page-break-before: always">
+  <td valign="top">
+    <h1 style="font-family: sans-serif; font-size: 14px; line-height: 29px; font-weight: normal; margin: 0 0 11px 0; text-align: center;">{{t "Firefox Account verified. You're almost there." }}</h1>
+
+    <h1 class="primary" style="font-family: sans-serif; font-size: 21px; line-height: 21px; font-weight: normal; margin: 0 0 21px 0; text-align: center;">{{t "Next sync between your devices!"}}</h1>
+
+    <div style="text-align: center; margin-bottom: 20px;">
+      <img src="https://image.e.mozilla.org/lib/fe9915707361037e75/m/5/f9463f08-8831-49fb-bbc4-7b5072cb63be.png"
+           height="175"
+           width="238"
+           alt="Devices"
```

Comment:
@vbudhram I'm down to rename any of the files here, but since we have a `postVerifySecondary`, it seemed like this as a partial `postVerifyPrimary` used in `postVerify` and `postVerifyNonMobile` made sense, though honestly I think it'd make more sense to call the partial `postVerify` and the two templates `postVerifyPrimaryOnMobile` or something of the sort. I went with the `NonMobile` verbiage since Alex confirmed we can treat tablets like desktops here.

Additionally, I wondered if I should go ahead and remove "Devices" here as the alt text. It's better a11y, but we don't have alt text for any images I'm guessing so you don't see a flash of text?

Reply:
I agree, there is probably some partials optimization that could be done in here. If we used the `postVerify` template in this PR, I think we could spin the optimization stuff into its own issue.

> I wondered if I should go ahead and remove "Devices" here as the alt text

SGTM